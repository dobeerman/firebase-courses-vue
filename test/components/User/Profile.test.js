import { shallow } from '@vue/test-utils'
import Profile from '../../../src/components/User/Profile.vue'

describe('Profile.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(Profile)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
