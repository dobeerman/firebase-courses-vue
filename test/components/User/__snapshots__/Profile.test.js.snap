// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile.vue matches snapshot 1`] = `
<div>
  <p>The User Page</p>
</div>
`;
